helm/icees-api/templates/server-ingress.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    # This line will automatically generate a Let's Encrypt TLS certificate which will be stored in the secretName below. See https://cert-manager.io/docs/usage/ingress/
+    # This only works for DNS names in public zones like *.renci.org or *.apps.renci.org. See https://wiki.renci.org/index.php/Kubernetes_Cloud/Let%27s_Encrypt_Migration
+    nginx.ingress.kubernetes.io/whitelist-source-range: "0.0.0.0/0,::/0"      
```

Comment:
these params should be in values files as they will change between deployment types (dev/test/prod)